.github/copilot-instructions.md

```diff
@@ -147,14 +180,22 @@ We use the `GitHub` repository https://github.com/Deltares/Ribasim for issues an
 
 ## Testing Strategy
 
-### Julia Tests
-- Unit tests in `core/test/`
-- Integration tests in `core/integration_test/`
-- Regression tests in `core/regression_test/`
+### Test Generation
+Most tests use generated models from `python/ribasim_testmodels/`. Always run `pixi run generate-testmodels` before running tests.
+
+### Test Types
+- **Unit tests** - `core/test/` (Julia), `python/*/tests/` (Python)
+- **Integration tests** - `core/integration_test/`
+- **Regression tests** - `core/regression_test/`, marked with `@pytest.mark.regression`
 
-### Python Tests
-- Unit tests in `python/*/tests/`
-- Mark regression tests with `@pytest.mark.regression`
+### Running Single Tests
+```bash
+# Julia single test file
+julia --project=core --check-bounds=yes core/test/specific_test.jl
```

Comment:
This won't work, because a test environment needs to be used rather than the core environment.

I don't know the right version right now, perhaps just leave it out.